src/components/EmojiSuggestions.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+
+import styles from '../styles/styles';
+import * as StyleUtils from '../styles/StyleUtils';
+import * as EmojiUtils from '../libs/EmojiUtils';
+import Text from './Text';
+import getStyledTextArray from '../libs/GetStyledTextArray';
+import AutoCompleteSuggestions from
+'./AutoCompleteSuggestions';
+
+const propTypes = {
+    /** The index of the highlighted emoji */
+    highlightedEmojiIndex: PropTypes.number,
+
+    /** Array of suggested emoji */
+    emojis: PropTypes.arrayOf(PropTypes.shape({
+        /** The emoji code */
+        code: PropTypes.string,
+
+        /** The name of the emoji */
+        name: PropTypes.string,
```

Comment:
These 2 can be `PropTypes.string.isRequired`, right? Additionally, let's document `types`

Reply:
Yes, I believe these can be marked as `isRequired`. I've also added some information about the `types`.